ls1intum/Artemis
# `Assessment`: Change naming for button when assessing a programming exercise for  the first time

### Checklist
#### General


#### Client

### Motivation and Context


The Button for starting an assessment, **in case you have not assessed a submission from this exercise before**, was a bit misleading and confusing. Changing the label to a more fitting description would be better. 

### Description

Change "Start participating in the exercise" to "Start manual assessment"

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Students
- 1 Programming Exercise 

1. Log in to Artemis
2. Create a new Programming exercise
3. Log in as a student and submit exercise
4. Go to the Assessment Dashboard of the exercise as an instructor (Important: This instructor should not assessed other submissions in the same exercise before, otherwise the button will not be shown)

### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage







### Screenshots


![image](https://github.com/ls1intum/Artemis/assets/65083942/1b3d07a3-4c07-44d8-88c8-d12a99aa67ab)
